.github/workflows/publish_pypi.yml

```diff
@@ -79,40 +78,37 @@ jobs:
         with:
           python-version: 3.8
 
-      - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==2.12.3
-
       - name: Clone pybind11 repo (no history)
-        run: git clone --depth 1 --branch v2.10.4 https://github.com/pybind/pybind11.git
+        run: git clone --depth 1 --branch v2.11.1 https://github.com/pybind/pybind11.git
 
-      - name: Install SUNDIALS on macOS
+      # sometimes gfortran cannot be found, so reinstall gcc just to be sure
+      - name: Install SuiteSparse and SUNDIALS on macOS
         if: matrix.os == 'macos-latest'
         run: |
-          # https://github.com/actions/virtual-environments/issues/1280
-          rm -f /usr/local/bin/2to3*
-          rm -f /usr/local/bin/idle3*
-          rm -f /usr/local/bin/pydoc3*
-          rm -f /usr/local/bin/python3*
           brew update
+          brew install gcc gfortran libomp graphviz openblas
           brew reinstall gcc
```

Comment:
do we still need this reinstlal of gcc since we install on the line above?

Reply:
About this: I think it would be wise to keep it. I noticed that three workflow runs on this branch passed without it, but one of them started failing randomly afterwards with the error message that `gfortran` wasn't found, which comes bundled with `gcc`. I think something is up on the GitHub Actions runners' side rather than Homebrew's, but I haven't filed a bug report on either place upstream yet